doc/UsersGuide/source/solving.rst

```diff
@@ -73,6 +73,29 @@ solver and furthermore it has the following IDA specific flags:
 :ref:`idaNonLinConvCoef <simflag-idaNonLinConvCoef>`,
 :ref:`idaMaxErrorTestFails <simflag-idaMaxErrorTestFails>`.
 
+
+.. _sundials_cvode :
+
+CVODE
+~~~~~
+
+The CVODE solver is part of sundials: SUite of Nonlinear and
+DIfferential/ALgebraic equation Solvers :cite:`Hindmarsh:2005`.
+CVODE solves initial value problems for ordinary differential equation (ODE)
+systems with variable-order, variable-step multistep methods.
+
+In OpenModelica CVODE uses a combination of Backward Differentiation
+Formulas (BDFs), with order varying between 1 and 5, and a Modified Newton
+iteration with fixed Jacobian as default.
+This setting is advised for stiff problems and therefore chooses by default.
+For non-stiff problems a combination of Adams-Moulton formulas, with the order
+varying between 1 and 12, and functional iteration as non-linear solver can
```

Comment:
Not sure exactly what you mean here

Reply:
````
For non-stiff problems an Adams-Moulton formula (varying order 1 to 12) is used as linear multi-step method
together with a fixed-point iteration as non-linear solver method.
```
Better?

Reply:
It's a nonlinear solver internal to cvode that is not provided by us, right?

Reply:
Correct